### PR TITLE
P1 features

### PR DESCRIPTION
**Pull request by msdeshmukh009**
### Features added 
- Timer: The user will have 1min 30Sec
- Quiz quiz: The user can quit the quiz.
- Create your quiz: The user can create their own quiz and add their own questions.
- Refactored user profile to show user's quiz-related data.

### CODE REVIEW

File: src/pages/profile/Profile.tsx [hunk 1]
```diff
@@ -18,11 +24,14 @@ const Profile = () => {
       try {
         setIsLoading(true);
         const userProfile = await getUserDocument(uid);
+        console.log(userProfile);
```
Comment (Chakravarthi0) [inline src/pages/profile/Profile.tsx]: This console.log can be removed, if not needed

File: src/pages/profile/profile.css [hunk 2]
```diff
@@ -46,8 +45,35 @@
...
-  align-self: flex-start;
+.quiz-stats {
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+  border: 1px solid var(--secondary-color);
+  /* background-color: var(--secondary-color);
+  color: var(--primary-color); */
+  border-radius: 5px;
+  /* justify-content: space-between; */
```
Comment (Chakravarthi0) [inline src/pages/profile/profile.css]: You can remove the comments which are not required

File: src/utils/firebaseHelpers.ts [hunk 3]
```diff
@@ -82,4 +98,62 @@ const getQuiz = async (quizId: string) => {
...
+  quizDescription: string;
+  quizCategory: string;
+  quizImage: string;
+}) => {
+  try {
+    const quizRef = collection(db, "quizzes");
+    const res = await addDoc(quizRef, { ...quizData, quizStatus: "available" });
+    return res.id;
+  } catch (err) {
+    return "something went wrong";
```
Comment (Chakravarthi0) [inline src/utils/firebaseHelpers.ts]: In ```addQuestion``` function, youre using ```getErrorMessage``` to get the error message, you can use that function here as well

**Review by Chakravarthi0 — APPROVED** [review]
code looks lean.
there is a minor bug, create quiz button is disabled when default category is selected:
image
Apart from that, app is working as expected.
great work!

**Review by pritam-kr — APPROVED** [review]
Everything working as good. I check each and every features all working fine.
- code is also clean 
- all services and file folders are abstracted.
- Good to merge

File: src/pages/profile/Profile.tsx (hunk 1 above)
Comment (msdeshmukh009) [reply src/pages/profile/Profile.tsx]: done

File: src/pages/profile/profile.css (hunk 2 above)
Comment (msdeshmukh009) [reply src/pages/profile/profile.css]: done

**Pull request merged**